e2e/cypress/integration/02_global_filter.spec.js

```diff
@@ -0,0 +1,76 @@
+describe('global projects filter', () => {
+  const nonAdminUsername = "cypressTestUserNonAdmin"
+  const proj1  = "cypress-project-1"
+  const proj2  = "cypress-project-2"
+  const pol_id = "cypress-policy"
+
+  before(() => {
+    cy.adminLogin('/').then(() => {
+      let admin = JSON.parse(localStorage.getItem('chef-automate-user'))
+
+      cy.createProject(admin.id_token, proj1)
```

Comment:
I've pulled these into helpers because I think it reads better and I imagine they'll be helpful later. But, you can accuse me of YAGNI and I'll remove them and put the code inline if that seems better.